Makefile

```diff
@@ -67,7 +67,8 @@ api-docs: gen-crd-api-reference-docs kustomize
 	set -e ;\
 	TMP_DIR=$$(mktemp -d) ; \
 	$(KUSTOMIZE) build config/ -o $$TMP_DIR/crd-output.yaml ;\
-	$(API_REF_GEN) crdoc --resources $$TMP_DIR/crd-output.yaml --output docs/docs/api.md --template frontmatter.tmpl;\
+	$(API_REF_GEN) crdoc --resources $$TMP_DIR/crd-output.yaml --output docs/docs/api.md --template markdown.tmpl;\
+	printf "\n" >> docs/docs/api.md ;\
```

Comment:
markdown.tmpl needs to be used to fix broken links

Need to add a newline at the end of api.md to appease the GitHub action which checks for newlines at the end of files.